### ISSUE TITLE
Improvement Idea: Import Multiple Profiles At Once

### ISSUE THREAD

**Issue by filipgoc** [Ultimaker/Cura#1304]
## Now: 
Only one quality/ material profile can be imported at a time. 
That makes the importing business quite a pain if one has more than one profile to import. 
![image](https://cloud.githubusercontent.com/assets/16858474/21672037/ce99fb48-d31f-11e6-8ed0-2edca4cb0ad5.png)

## Desired: 
Select as many quality profiles/ material profiles (even a folder maybe? oh my, let's not get too risky) as desired and import all of them. 
![image](https://cloud.githubusercontent.com/assets/16858474/21672039/d405f49c-d31f-11e6-80e8-eec7dd32a8e3.png)

**Comment by nallath** [comment]
I've logged this feature in our internal tracker. For our reference; CURA-3207

**Comment by ianpaschal** [comment]
This has been marked `Won't Fix` internally.

**Comment by Ghostkeeper** [comment]
It was deemed not important enough because it will save some users a few minutes only once per installation.
